site/docs/components/multiline-input/examples.mdx

```diff
@@ -0,0 +1,124 @@
+---
+
+<LivePreviewControls>
+<LivePreview componentName="multiline-input" exampleName="Primary" >
+
+### Primary
+
+Values entered into the Multiline Input are set to wrap based on the width of the component. Initially, the component is empty. Once it has focus it accepts text across multiple lines. By default, the component presents three rows of text at a time.
+
+If entering more than through rows of text, the user can review the entered value using keyboard controls.
+
+- Multiline Input has two variants: "primary" and "secondary". The primary variant is default. The choice between primary and secondary variants should be considered alongside the background color it sits upon. For recommendations on variant choice, see the [Forms pattern](/salt/patterns/forms).
+
+</LivePreview>
+
+<LivePreview componentName="multiline-input" exampleName="Secondary">
+
+### Secondary
+
+Secondary variant is an effective option if included as part of a data-dense or compact interface, or within a form with multiple columns or sections.
+
+The choice between primary and secondary variants should be considered alongside the background color it sits upon.
+
+- For recommendations on variant choice, see the [Forms pattern](/salt/patterns/forms).
+
+</LivePreview>
+<LivePreview componentName="multiline-input" exampleName="Disabled">
+
+### Disabled
+
+Multiline Input can be set to a disabled state, with no resultant action when the user interacts with it. Showing it in a disabled state allows the user to be aware of its presents or an entered value while restricting their ability to edit it.
+
+</LivePreview>
+<LivePreview componentName="multiline-input" exampleName="Readonly">
+
+### Readonly
+
+Multiline Input can be set to a read-only state. The entered text cannot be edited in this state, but it can be highlighted and copied.
+
+Read-only fields are not focusable since they are treated as a static representation rather than a form control.
+
+Use the read-only state when the value is necessary for the users’ flow or current task but cannot be edited, for instance when a users permissions are restricted.
+
+When setting Multiline Input to a read-only state, ensure to also disable any nested interactive adornments, such as buttons.
+
+</LivePreview>
+<LivePreview componentName="multiline-input" exampleName="Bordered">
+
+### Bordered
+
+To style Multiline Input with a full border set the `bordered` prop to `true`.
+
+- We recommend this styling when the field uses the same fill color as the background (i.e., a primary fill color on a primary background).
+
+</LivePreview>
+<LivePreview componentName="multiline-input" exampleName="NumberOfRows">
+
+### Number Of Rows
+
+The Multiline Input shows three rows of text by default. This can be adjusted, using the ‘rows’ prop, to provide a larger input allowing the user to review more of the entered text without the need for further interactions. This example demonstrates a Multiline Input customised to show four rows.
```

Comment:
```suggestion
The Multiline Input shows three rows of text by default. This can be adjusted, using the `rows` prop, to provide a larger input allowing the user to review more of the entered text without the need for further interactions. This example demonstrates a Multiline Input customised to show four rows.
```